app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -38,6 +38,15 @@ examples:
       suggestion_link_text: "You can now apply for a permit online."
       suggestion_link_url: "/permit"
 
+  with_hide:
+    description: |
+      This example is for when we want to hide by default and display to the user based on some logic,
+      either in the backend or with Javascript.
```

Comment:
nitpick: why is this on a different line?

Reply:
It doesn't affect the output but just to keep it within a useful width for editing (simply a style guide thing which we don't have defined for MD specifically but is what we have for the rest of our code syntaxes)

Reply:
I'm not too sure, personally. Breaking it presumes dev screen size and I _think_ most editors can wrap text anyway for easy of editing. Do you have any examples of where else we do this?

Caveat to this comment that I am absolutely not going to block this going out over a disagreement of MD styling. If you've gotta merge then merge away.

Reply:
I know most editors will soft wrap but the general standard otherwise is ~80 chars and it just looks tidier... ¯\_(ツ)_/¯ 

Thank you again for your review!